source/Halibut.Tests/TestServices/SupportedServices.cs

```diff
@@ -92,5 +92,10 @@ public string Ambiguous(string a, Tuple<string, string> b)
         {
             return "Hello tuple";
         }
+
+        public MapLocation GetLocation(MapLocation loc)
+        {
+            return new MapLocation { Latitude = loc.Longitude, Longitude = loc.Latitude };
```

Comment:
Latitude = loc.Longitude?

Reply:
Yeah, that's the "change" for the round-trip verification - I'll add a comment.